common/README.md

```diff
@@ -0,0 +1,111 @@
+near-blank-project
+==================
+
+This [NEAR] app was initialized with [create-near-app]
+
+
+Quick Start
+===========
+
+To run this project locally:
+
+1. Prerequisites: Make sure you have [Node.js] ≥ 12 installed
+2. Install dependencies: `npm install`
+3. Run the local development server: `npm run dev` (see `package.json` for a
+   full list of `scripts` you can run with `npm`)
```

Comment:
We have different prereqs for asc and rust (rust requires some extra installation steps). Does it still make sense to put this in common?

Reply:
Would we need different steps if we implemented #374?

Reply:
This is related to #373, right? 

My thought was that the READMEs seemed 95% similar, so it becomes quite a burden to maintain two. If necessary, we can, but I would rather prioritize #374 and keep the READMEs maintainable, if that makes sense to you, @janedegtiareva

Reply:
> The backend code lives in the `/assembly` folder
different for Rust
> The smart contract code gets tested with the [asp] command
different for Rust

Reply:
Yep, three things are different right now. These are currently being replaced when a Rust app is created; see https://github.com/near/create-near-app/pull/389/files#diff-168726dbe96b3ce427e7fedce31bb0bcR93-R95

I think this seems more maintainable than having N*M duplicate READMEs, where `N=number of frontend templates` and `M=number of contract languages`, but that's exactly what I want feedback on

(Would we need N*M? Maybe we only need the same number as contract languages)

Pros/cons for current approach:

* pro: no one will accidentally change one kind of README while leaving the others outdated
* con: the `replaceInFiles` calls could result in unexpected typos/errors in the Rust version
* con: someone might add AssemblyScript-specific stuff to the README and forget to add new `replaceInFiles` calls, resulting in errant Rust READMEs

When thinking about this, I want to aim for the implementation that results in the easiest maintainability for 90% of changes. I want something that's mostly fool-proof, that could be maintained by a community of volunteers.

Am I weighting the "pro"  too heavily?

Reply:
I think that the pro might be weighed a little too heavily. I wouldn't think to update a set of functions if I were just updating READMEs. I think it's more likely to confuse than help. It makes sense to me to have just two readmes specific to the backend code that explain the project, and then you can have a separate readme for how to run the frontend in `src/` assuming they are necessary. Sure it means more READMEs, but they should be specific enough to each. This seems like one step into over-optimization.